<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>工具栏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset_all">
        <property name="text">
         <string>重置所有设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset_font">
        <property name="text">
         <string>重置字体设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset_window">
        <property name="text">
         <string>重置窗口设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_minus">
        <property name="text">
         <string>缩小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_plus">
        <property name="text">
         <string>放大</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_de_all">
        <property name="text">
         <string>注销所有设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_de_font">
        <property name="text">
         <string>注销字体设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_de_window">
        <property name="text">
         <string>注销窗口设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_dc_all">
        <property name="text">
         <string>解绑所有读取事件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_dc_font">
        <property name="text">
         <string>解绑字体读取事件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_dc_window">
        <property name="text">
         <string>解绑窗体读取事件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0,8">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>左
侧
工
具
栏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_2">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>工
具
栏
展
开</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_3">
        <property name="plainText">
         <string>文本编辑器</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单栏 1  </string>
    </property>
    <addaction name="separator"/>
    <addaction name="action1"/>
    <addaction name="actionaaaa"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>菜单栏 2  </string>
    </property>
    <addaction name="action2"/>
    <addaction name="actionbbb"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>菜单栏 3  </string>
    </property>
    <addaction name="actionc"/>
    <addaction name="actioncccc"/>
    <addaction name="actionddd"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>菜单栏 4  </string>
    </property>
    <addaction name="actionllll"/>
    <addaction name="actionla"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionaaaa">
   <property name="text">
    <string>aaaa</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actionbbb">
   <property name="text">
    <string>bbb</string>
   </property>
  </action>
  <action name="actionc">
   <property name="text">
    <string>c</string>
   </property>
  </action>
  <action name="actioncccc">
   <property name="text">
    <string>cccc</string>
   </property>
  </action>
  <action name="actionddd">
   <property name="text">
    <string>ddd</string>
   </property>
  </action>
  <action name="actionllll">
   <property name="text">
    <string>llll</string>
   </property>
  </action>
  <action name="actionla">
   <property name="text">
    <string>la</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
